src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.cxx

```diff
@@ -2047,7 +2060,7 @@ PlusStatus vtkPlusCapistranoVideoSource::InternalApplyImagingParameterChange()
     this->ImagingParameters->GetTimeGainCompensation(tgcVec);
     double tgc[3] = { tgcVec[0], tgcVec[1], tgcVec[2] };
 
-    if (this->SetGainPercent(tgc) == PLUS_SUCCESS)
+    if (this->SetGainPercentDevice(tgc) == PLUS_SUCCESS)

```

Comment:
this should probably go in a different commit as it's somewhat separate from the command addition? ("BUG: Capistrano TGC not updated through remote commands" or something like that)